addon/components/bourbon-search-select-field.js

```diff
@@ -90,8 +94,12 @@ export default Component.extend(SelectMixin, {
     // e.keyCode 13 is for 'Enter'
     if (e.keyCode === 13) {
       e.preventDefault();
+      if (this.get('searchList') !== 'No results found.' &&  this.get('searchList').length > 0) {
```

Comment:
Isn't `searchList` an array? I'm assuming you ahve to do something like `this.get('searchList')[0]` or something?

Reply:
Also if it's grouped or has a label path it'll have to be handled differently i think

Reply:
It was meant to return the first option if there was a valid list of options and someone presses enter.  I found a better way to represent what I was intending...hopefully.  :)